### DISCUSSION TITLE
How to bind only saves directories?

### DISCUSSION THREAD

**Discussion by MaksimShakavin** [wolveix/satisfactory-server#328] [Q&A] [answered]
I want to store my saves, blueprints, backups, and all other files that don’t require fast disk access on my NAS using NFS. For everything else, I plan to use a fast SSD on the host itself.

In various examples, I’ve noticed recommendations to bind the entire `/config` directory. However, it seems to contain server internal files, and I’m not sure how well it would perform with NFS.

So my questions are:

1. Which directory is safe to put on my NAS while keeping the essential server files on the host?
2. I found this directory:
```GAMESAVESDIR="/home/steam/.config/Epic/FactoryGame/Saved/SaveGames`, but it only contains a couple of outdated autosaves.
Should I bind this directory as well?

**Comment by wolveix** [comment, marked as answer]
Hi, no you're overthinking this.

If you absolutely want to split out the saves directory, simply modify your Docker config to bind `/config` to your existing path`, but then also bind `/config/saves` to your separate path.

**Comment by seigneurao** [comment]
Hi @MaksimShakavin .

I don't know what's your exact purpose here, but supposing you have the same intent I had, which is to have a backup (or, even better, several) in case anything happens to the remote host, just wanted to mention that I wrote a short script which tar.gzes the /server directory, then stores it on my NAS incrementally with rsync. It is croned twice a day.
